Assets/Scripts/Interactable/ShipFastTravel.cs

```diff
@@ -21,28 +21,27 @@ public class CheckFastTravel : MonoBehaviour
     public GameObject MainShip;
     public GameObject ShipParent;
 
-    public GameObject[] PlayersInScene;
-
     #endregion
 
     #region Private Variables
 
     private int _count = 0;
 
-    private FastTravelUI _travelDestination;
+    private FastTravelNPC _travelDestination;
```

Comment:
Rename to `_fastTravelNPC` since this is the NPC reference and not the position of the destination